iOS-Settings/Sources/App/SettingsController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ViewController.swift
+//  iOS-Settings
+//
+//  Created by Kuanysh al-Khattab Auyelgazy on 02.03.2023.
+//
+
+import UIKit
+import SnapKit
+
+class SettingsController: UIViewController {
+
+    private var settings = [[Setting]]()
```

Comment:
make it private let setting: Settings

implement init:
init() {
    settings = [Setting.networkSection, Setting.soundsSection, Setting.generalSection]
}